src/app/core/campaign/campaign-store.service.ts

```diff
@@ -50,12 +50,9 @@ export class CampaignStoreService {
               (acc, day) => {
                 // get the day's allocationPreview from the allocationPreview state
                 day.allocationPreview =
-                  state.allocationPreview &&
-                  state.allocationPreview[`${flightId}`] &&
-                  state.allocationPreview[`${flightId}`][availability.zone] &&
-                  state.allocationPreview[`${flightId}`][availability.zone].allocations &&
-                  state.allocationPreview[`${flightId}`][availability.zone].allocations[day.startDate] &&
-                  state.allocationPreview[`${flightId}`][availability.zone].allocations[day.startDate].goalCount;
```

Comment:
for now we just grab the allocation preview from the store's entities here instead

Reply:
This code is fragile and will be adjusted in the refactor. I just pushed up another commit to address this bug where the allocation preview thinks it should be showing but there aren't value (because false >= 0 is what the component was comparing.)